### PR TITLE
Space Travelers' Hub project submission PR

### PR DESCRIPTION
**Pull request by mahbubul14**
## [Project requirements](https://github.com/microverseinc/curriculum-react-redux/blob/main/group-project/project_space_travelers_hub.md):

* Config & basic setup
* Redux: Fetch data and update Redux store
* Render UI: lists
* Redux: Write actions and reducers for booking rockets/dragons and joining missions
* Redux: Write actions and reducers for canceling rockets/dragons and leaving missions
* Render UI: conditional components rendering
* Render UI: My Profile section
* Technical set up

**Note:** The contents displayed on the Rocket's page are fetched from a Rocket API. When a user clicks the "Reserve rocket" button, actions are dispatched to update the store and the reserved rocket shows a "Reserved" badge and "Cancel reservation" button instead of the default "Reserve rocket"

The contents displayed on the Missions' page are fetched from a Missions API. When a user clicks the "Join Mission" button, actions are dispatched to update the store, the join mission button changes to "Leave Mission" and the corresponding badge for that mission shows "Active Member". On the other hand, when the "Leave Mission" button is clicked, the button changes to "Join Mission" and the corresponding badge shows "Not A Member"

The Profile Page displays the Name of All Reserved Rockets and the Joined Missions.

### CODE REVIEW

File: src/component/Nav.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+        <NavLink to="/">Rockets</NavLink>
+      </li>
+      <li className="mr-2">
+        <NavLink to="/missions">Missons</NavLink>
+      </li>
+      <li className="mr-2">
+        <NavLink to="/my-profile">My Profile</NavLink>
+      </li>
+    </ul>
+  </nav>
```
Comment (omar-labana) [inline src/component/Nav.js:22]: - [ ] Please make sure that the currently active page should be indicated with an underline.

File: src/component/Missions.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    <div className="missions-container">
+      <span><b>Mission</b></span>
+      <span><b>Description </b></span>
+      <span><b>Status</b></span>
+      <span>{emptyHeader}</span>
+      {missionsData.map((item) => (<SingleMission key={item.mission_id} data={item} />
+      ))}
+    </div>
+  );
+};
```
Comment (omar-labana) [inline src/component/Missions.js:23]: - [ ] Data should not be added to the store on every re-render (i.e. when changing views / using navigation). Currently a new request is made when I switch between pages on the mission's page (check `network` tab from `dev tools`)

File: src/component/Missions.js [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+  }, []);
+  const emptyHeader = '';
+  return (
+    <div className="missions-container">
+      <span><b>Mission</b></span>
+      <span><b>Description </b></span>
+      <span><b>Status</b></span>
+      <span>{emptyHeader}</span>
+      {missionsData.map((item) => (<SingleMission key={item.mission_id} data={item} />
+      ))}
```
Comment (omar-labana) [inline src/component/Missions.js:20]: - [ ] The `missions` should use the React conditional rendering syntax as a fall back if the data from the API wasn't retrieved successfully.

**Review by omar-labana — CHANGES_REQUESTED** [review]
Hi team, 

Excellent work so far! 👌
You have implemented almost all the required features. ✔️ 
There are some cases that you need to adjust but you are almost done! 😀

## Required Changes ♻️

- [x] Kindly remove the README screenshot to a `docs` directory instead of the `src` directory.

- [x] _Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 4]
```diff
@@ -1 +1,92 @@
...
+
+- GitHub: [@mahbubul14](https://github.com/mahbubul14)
+- Twitter: [@twitterhandle](https://twitter.com/mahbubul_14)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/mahbubul-alam-20595/)
+
+
+👨🏿 **Akintoye Damilare**
+
+- GitHub: [@dami1080](https://github.com/dami1080)
+- LinkedIn: []()
```
Comment (bimbolabuari) [inline README.md:76]: [Optional] Let's add your LinkedIn URL here and not leave it empty, this makes recruiters see your readme as a professional one 👍

**Review by bimbolabuari — APPROVED** [review]
# Project Approved 🚀

Hi Team 🥇,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**